### ISSUE TITLE
Make _googleDns public

### ISSUE THREAD

**Issue by Solido** [matanlurey/cynic#3]
It's useful to have googleDns public so I can call it on top of my own servers

```dart
static const servers = const [
    Service.googleDns, // instead of const Service.forIp('8.8.8.8', name:'google'),
    const Service.forIp('192.168.66.221', name:'api'),
    const Service.forIp('192.168.66.222', name:'auth'),
    const Service.forIp('192.168.66.223', name:'cold'),
  ];
```

**Comment by matanlurey** [comment]
OK, I like this idea.

Want to send a PR? @Solido

**Comment by Solido** [comment]
I've started to work on the widget so I'll be happy share code/idea with review.

**Comment by matanlurey** [comment]
Closed by #6.
